wiki/Tournaments/OMIC/2022_4K/id.md

```diff
@@ -0,0 +1,420 @@
+| Pembuat Map | ::{ flag=ID }:: [Ainer](), ::{ flag=ID }:: [Angki](https://osu.ppy.sh/users/9038483), ::{ flag=ID }:: [Antipole](https://osu.ppy.sh/users/17258072), ::{ flag=ID }:: [FelixSpade](https://osu.ppy.sh/users/2651304), ::{ flag=ID }:: [Reyi](https://osu.ppy.sh/users/13385865), ::{ flag=GB }:: [TingMomentum](https://osu.ppy.sh/users/12489832) |
+| Pencoba Map | ::{ flag=PH }:: [4Kenn](https://osu.ppy.sh/users/20496815), ::{ flag=DE }:: [ERA Punish](https://osu.ppy.sh/users/10615367), ::{ flag=ID }:: [Fyl](https://osu.ppy.sh/users/10069307), ::{ flag=IT }:: [- Pixie -](https://osu.ppy.sh/users/13878539), ::{ flag=PH }:: [Silicosis](https://osu.ppy.sh/users/15548876), ::{ flag=NL }:: [Toxic Scent](https://osu.ppy.sh/users/12599154) |
+| Penyiar | ::{ flag=ID }:: [Tix](https://osu.ppy.sh/users/11421465), ::{ flag=ID }:: [zutu](https://osu.ppy.sh/users/11184912) |
+| Wasit | ::{ flag=DZ }:: [Demon-kira](https://osu.ppy.sh/users/20232838), ::{ flag=ID }:: [\[-Elaina-\]](https://osu.ppy.sh/users/12532297), ::{ flag=ID }:: [ElinLYPK](https://osu.ppy.sh/users/5198950), ::{ flag=ID }:: [Kurami\_San](https://osu.ppy.sh/users/8867495), ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515), ::{ flag=IN }:: [shreya7](https://osu.ppy.sh/users/16676388), ::{ flag=CH }:: [Sinsa92](https://osu.ppy.sh/users/13378002) |
+| Komentator | ::{ flag=ID }:: [2ndlegend](https://osu.ppy.sh/users/7621604), ::{ flag=ID }:: [AZKiFanboy](https://osu.ppy.sh/users/5179764), ::{ flag=ID }:: [EstaticStatisIO](https://osu.ppy.sh/users/17237364), ::{ flag=ID }:: [Oofyxl](https://osu.ppy.sh/users/20599160), ::{ flag=ID }:: [RayhanDelanova](https://osu.ppy.sh/users/17898143), ::{ flag=ID }:: [Revv-](https://osu.ppy.sh/users/12424909) |
+| Desainer | ::{ flag=ID }:: [Yui Funami](https://osu.ppy.sh/users/14752579) |
+| Pengelola Sheet | ::{ flag=GB }:: [chromb](https://osu.ppy.sh/users/10238680), ::{ flag=ID }:: [KotoriPana](https://osu.ppy.sh/users/6866937), ::{ flag=ID }:: [Sies](https://osu.ppy.sh/users/6491991) |
+| Editor Wiki | ::{ flag=ID }:: [KotoriPana](https://osu.ppy.sh/users/6866937), ::{ flag=ID }:: [Sies](https://osu.ppy.sh/users/6491991) |
+| Sponsor | ::{ flag=ID }:: [Wowcake](https://osu.ppy.sh/users/16121851) |
+
+## Daftar Tautan Terkait
+
+- **[Master Spreadsheet](https://s.id/OMIC2022MasterSheet)**
+- [Topik diskusi](https://osu.ppy.sh/community/forums/topics/1581867?n=1)
+- [Siaran langsung](https://www.twitch.tv/osumic_live)
+- [Server Discord](https://discord.gg/pfjjrbtRCh)
+- [Bagan Challonge](https://challonge.com/omic2022)
+
+## Daftar Peserta
+
+### Kualifikasi
```

Comment:
don't think this line is needed tbh, could just delete it altogether

Reply:
will be applied in a separate commit. Thank you.